python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -108,8 +115,35 @@ public void testEntrypoint(
             response = ProgressEvent.defaultFailureHandler(e, HandlerErrorCode.InternalFailure);
         } finally {
             writeResponse(outputStream, response);
+            outputStream.close();
+        }
+    }
+    {% else %}
+    public static void main(String[] args) throws IOException {
+        if (args.length != 1){
+            System.exit(1);
+        }
+        final String outputFile = (UUID.randomUUID().toString() + ".txt");
+        try(FileOutputStream output = new FileOutputStream(outputFile)){
+            try(InputStream input=IOUtils.toInputStream(args[0],"UTF-8")){
+                new ExecutableHandlerWrapper().handleRequest(input, output);
+                output.flush();
+            }
```

Comment:
is there a way to print out the output using the output stream instead of saving the file then reading it again?

Reply:
To get the results from a jar, you use either system out or can read from a file that the jar wrote to. I chose system out because that doesn't require the consumer to also read from a file (which also includes mounting a volume in an image so that they can access the file from the container as well). However, standard out also includes the logs so I need some way to differentiate the logs and the handler result. I can't just split it by newline and take the last element in the list because there may be new lines in the actual handler result. I ended up doing this way in order to be able to use a regex in order to grab the response in the cfn cli